src/app/fyle/view-team-report/view-team-report.page.ts

```
@@ -353,13 +353,18 @@ export class ViewTeamReportPage {
-      const rpAmount = this.humanizeCurrency.transform(report.amount, report.currency, false);
+      const rpAmount = this.exactCurrency.transform({
+        value: report.amount,
+        currencyCode: report.currency,
+        skipSymbol: false,
+      });
       const flaggedExpensesCount = expenses.filter((expense) => expense.is_policy_flagged).length;
       const popover = await this.popoverController.create({
         componentProps: {
           flaggedExpensesCount,
           title: 'Approve Report',
           message: report.num_expenses + ' expenses of amount ' + rpAmount + ' will be approved',
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)